cpp/oneapi/dal/algo/svm/backend/utils/model.hpp

```diff
@@ -0,0 +1,61 @@
+/*******************************************************************************
```

Comment:
Let's not create even more directories in backend, I'd suggest to rename it to `interop_model.hpp` and put to `svm/backend`

Reply:
Applied